packages/platform/src/HttpApiSwagger.ts

```diff
@@ -25,16 +57,19 @@ export const layer = (options?: {
 <head>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <title>${spec.info.title} Documentation</title>
+  <title>${htmlEscape(spec.info.title)} Documentation</title>
   <style>${internal.css}</style>
 </head>
 <body>
   <div id="swagger-ui"></div>
+  <script id="swagger-spec" type="application/json">
```

Comment:
What's the purpose of moving it to a seperate script tag?

Reply:
IIRC a type="application/json" tag is never executed, so we don't accidentally run any code